autogpt/memory_management/summary_memory.py

```diff
@@ -0,0 +1,97 @@
+from typing import Dict, List, Tuple
+
+from autogpt.config import Config
+from autogpt.llm.llm_utils import create_chat_completion
+
+cfg = Config()
```

Comment:
nit: Pass this in from the agent if possible to help the agent encapsulation project. 
CC: @collijk

Reply:
Doing this gave me a circular dependency - I'll have to check how this is being done later